src/Servers/Kestrel/Core/src/Internal/Http/ChunkWriter.cs

```diff
@@ -44,6 +36,30 @@ public static int BeginChunkBytes(int dataCount, Span<byte> span)
             return count;
         }
 
+        internal static int GetBeginChunkByteCount(int count)
```

Comment:
I'd prefer if this called into the other GetBeginChunkByteCount

Reply:
I definitely intended to do that, must have missed it.